src/Modules/CalcPerform.lua

```diff
@@ -674,8 +674,21 @@ local function doActorAttribsPoolsConditions(env, actor)
 		modDB:ScaleAddList({ value.mod }, calcLib.mod(modDB, nil, "BuffEffectOnSelf", "ShrineBuffEffect"))
 	end
 
-	output.ChaosInoculation = modDB:Flag(nil, "ChaosInoculation")
 	-- Life/mana pools
+	local lowLifePerc = modDB:Sum("BASE", nil, "LowLifePercentage")
+	if lowLifePerc > 0 then
+		data.misc.LowLifePercentage = lowLifePerc
+	else
+		data.misc.LowLifePercentage = 50.0
+	end
+	local fullLifePerc = modDB:Sum("BASE", nil, "FullLifePercentage")
+	if fullLifePerc > 0 then
+		data.misc.FullLifePercentage = fullLifePerc
+	else
+		data.misc.FullLifePercentage = 100.0
+	end
```

Comment:
Don't change things in the data file during calculation.  That consists of static data that is loaded on start-up.

Reply:
@Wires77 how do I pass non-data file values into `ConfigOptions.lua`?

Reply:
Grr, you're right, you can't.  This is fine then, even if I hate it from a clean code perspective 🥲 It's just not worth writing a bunch of code to bring build data into the configOptions